src/components/graph/menus/create-node-menu.js

```diff
@@ -14,6 +14,43 @@ import { useSelector } from 'react-redux';
 import { CopyType } from '../../network-modification-tree-pane';
 import { NestedMenuItem } from 'mui-nested-menu';
 import ChildMenuItem from './create-child-menu-item';
+import { CustomDialog } from '../../utils/custom-dialog';
+
+export const NodeActions = {
+    REMOVE_NODE: 'REMOVE_NODE',
+    REMOVE_SUBTREE: 'REMOVE_SUBTREE',
+    NO_ACTION: 'NO_ACTION',
+};
+
+export const getNodeChildren = (treeModel, sourceNodeIds, allChildren) => {
```

Comment:
All pptional Chainings (var?) are really necessary ?

Reply:
Good catch, not all of them.Done.